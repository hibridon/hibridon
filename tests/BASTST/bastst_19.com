inp=bastst_19.inp
out=bastst_19.out
run
save=save_19.inp