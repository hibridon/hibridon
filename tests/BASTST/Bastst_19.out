      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** 2SG|2PI_ system parameters:
 NTERM  =   4        ISYM   =   1        ISA    =   1        IGUSG  =   1
 NMAXSG =   5        NPARSG =   2        IGUPI  =   1        NPARPI =   2
 NUMVPI =   1        JMAX(0)=   5
 ESIG   =  200.0     BSIG   = 0.8000     DSIG   = 2.0000E-03 GSR    = 0.2000    
 EPI(0) =  100.0     BPI(0) = 0.8000     DPI(0) = 2.0000E-03 ASO(0) = 1.0000E-02
 P(0)   = 1.0000E-02 Q(0)   = 1.0000E-02
 LAMMIN:    1   1   1   1
 LAMMAX:    1   1   1   1
 MPROJ:     0  -1  -1  -1
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= T
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT: -2; JOUT:  0   2
 INDOUT:    0   1
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-01-24 at 11:46
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 490a05d5957aa87d9c9668b1fd83ca5b6dbd246d
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Jan 24 14:45:15 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500

 **  2SIG + 2PI INT. COUPLING **
     2SIG:   g/u= 1  +/- =  1  s/a =  1
     2PI:    g/u= 1  s/a =  1

 ** CC **    RMU=  16.4700  E= 5000.00  JTOT= 20.5 JLPAR= 1

 2SIG PARAMETERS:
 NMAX     ESIG     BROT        DROT        GSR
    5    200.000   8.0000E-01  2.0000E-03  2.0000E-01

 2PI PARAMETERS:
  V JMAX     EVIB     BROT        DROT        ASO         P           Q
  0    5  100.000   8.0000E-01  2.0000E-03  1.0000E-02  1.0000E-02  1.0000E-02

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

   N   V   J    EPS   FI    L    EINT(CM-1)    C-1/2      C-3/2      CSIG
   1   0  0.5     1   3    20       99.207    0.00000    0.00000    1.00000
   2   0  0.5    -1   3    21      100.599    0.00000    0.00000    1.00000
   3   0  1.5     1   3    19      100.899    0.00000    0.00000    1.00000
   4   0  1.5     1   3    21      100.899    0.00000    0.00000    1.00000
   5   0  1.5    -1   3    20      103.635    0.00000    0.00000    1.00000
   6   0  1.5    -1   3    22      103.635    0.00000    0.00000    1.00000
   7   0  2.5     1   3    18      104.135    0.00000    0.00000    1.00000
   8   0  2.5     1   3    20      104.135    0.00000    0.00000    1.00000
   9   0  2.5     1   3    22      104.135    0.00000    0.00000    1.00000
  10   0  2.5    -1   3    19      108.119    0.00000    0.00000    1.00000
  11   0  2.5    -1   3    21      108.119    0.00000    0.00000    1.00000
  12   0  2.5    -1   3    23      108.119    0.00000    0.00000    1.00000
  13   0  3.5     1   3    17      108.819    0.00000    0.00000    1.00000
  14   0  3.5     1   3    19      108.819    0.00000    0.00000    1.00000
  15   0  3.5     1   3    21      108.819    0.00000    0.00000    1.00000
  16   0  3.5     1   3    23      108.819    0.00000    0.00000    1.00000
  17   0  3.5    -1   3    18      113.907    0.00000    0.00000    1.00000
  18   0  3.5    -1   3    20      113.907    0.00000    0.00000    1.00000
  19   0  3.5    -1   3    22      113.907    0.00000    0.00000    1.00000
  20   0  3.5    -1   3    24      113.907    0.00000    0.00000    1.00000
  21   0  4.5     1   3    16      114.807    0.00000    0.00000    1.00000
  22   0  4.5     1   3    18      114.807    0.00000    0.00000    1.00000
  23   0  4.5     1   3    20      114.807    0.00000    0.00000    1.00000
  24   0  4.5     1   3    22      114.807    0.00000    0.00000    1.00000
  25   0  4.5     1   3    24      114.807    0.00000    0.00000    1.00000
  26   0  4.5    -1   3    17      120.807    0.00000    0.00000    1.00000
  27   0  4.5    -1   3    19      120.807    0.00000    0.00000    1.00000
  28   0  4.5    -1   3    21      120.807    0.00000    0.00000    1.00000
  29   0  4.5    -1   3    23      120.807    0.00000    0.00000    1.00000
  30   0  4.5    -1   3    25      120.807    0.00000    0.00000    1.00000
  31   0  5.5     1   3    15      121.907    0.00000    0.00000    1.00000
  32   0  5.5     1   3    17      121.907    0.00000    0.00000    1.00000
  33   0  5.5     1   3    19      121.907    0.00000    0.00000    1.00000
  34   0  5.5     1   3    21      121.907    0.00000    0.00000    1.00000
  35   0  5.5     1   3    23      121.907    0.00000    0.00000    1.00000
  36   0  5.5     1   3    25      121.907    0.00000    0.00000    1.00000
  37   0  1.5    -1   1    20        0.007    0.50000    0.86603    0.00000
  38   0  1.5    -1   1    22        0.007    0.50000    0.86603    0.00000
  39   0  1.5     1   1    19        0.022    0.50239    0.86464    0.00000
  40   0  1.5     1   1    21        0.022    0.50239    0.86464    0.00000
  41   0  2.5    -1   1    19        3.159    0.57694    0.81678    0.00000
  42   0  2.5    -1   1    21        3.159    0.57694    0.81678    0.00000
  43   0  2.5    -1   1    23        3.159    0.57694    0.81678    0.00000
  44   0  2.5     1   1    18        3.209    0.57946    0.81500    0.00000
  45   0  2.5     1   1    20        3.209    0.57946    0.81500    0.00000
  46   0  2.5     1   1    22        3.209    0.57946    0.81500    0.00000
  47   0  3.5    -1   1    18        7.766    0.61175    0.79105    0.00000
  48   0  3.5    -1   1    20        7.766    0.61175    0.79105    0.00000
  49   0  3.5    -1   1    22        7.766    0.61175    0.79105    0.00000
  50   0  3.5    -1   1    24        7.766    0.61175    0.79105    0.00000
  51   0  3.5     1   1    17        7.871    0.61435    0.78904    0.00000
  52   0  3.5     1   1    19        7.871    0.61435    0.78904    0.00000
  53   0  3.5     1   1    21        7.871    0.61435    0.78904    0.00000
  54   0  3.5     1   1    23        7.871    0.61435    0.78904    0.00000
  55   0  4.5    -1   1    17       13.686    0.63168    0.77523    0.00000
  56   0  4.5    -1   1    19       13.686    0.63168    0.77523    0.00000
  57   0  4.5    -1   1    21       13.686    0.63168    0.77523    0.00000
  58   0  4.5    -1   1    23       13.686    0.63168    0.77523    0.00000
  59   0  4.5    -1   1    25       13.686    0.63168    0.77523    0.00000
  60   0  4.5     1   1    16       13.866    0.63439    0.77301    0.00000
  61   0  4.5     1   1    18       13.866    0.63439    0.77301    0.00000
  62   0  4.5     1   1    20       13.866    0.63439    0.77301    0.00000
  63   0  4.5     1   1    22       13.866    0.63439    0.77301    0.00000
  64   0  4.5     1   1    24       13.866    0.63439    0.77301    0.00000
  65   0  5.5    -1   1    16       20.726    0.64460    0.76452    0.00000
  66   0  5.5    -1   1    18       20.726    0.64460    0.76452    0.00000
  67   0  5.5    -1   1    20       20.726    0.64460    0.76452    0.00000
  68   0  5.5    -1   1    22       20.726    0.64460    0.76452    0.00000
  69   0  5.5    -1   1    24       20.726    0.64460    0.76452    0.00000
  70   0  5.5    -1   1    26       20.726    0.64460    0.76452    0.00000
  71   0  5.5     1   1    15       21.001    0.64747    0.76209    0.00000
  72   0  5.5     1   1    17       21.001    0.64747    0.76209    0.00000
  73   0  5.5     1   1    19       21.001    0.64747    0.76209    0.00000
  74   0  5.5     1   1    21       21.001    0.64747    0.76209    0.00000
  75   0  5.5     1   1    23       21.001    0.64747    0.76209    0.00000
  76   0  5.5     1   1    25       21.001    0.64747    0.76209    0.00000
  77   0  0.5    -1   2    21        0.030    1.00000    0.00000    0.00000
  78   0  0.5     1   2    20        0.000    1.00000    0.00000    0.00000
  79   0  1.5    -1   2    20        3.230   -0.86603    0.50000    0.00000
  80   0  1.5    -1   2    22        3.230   -0.86603    0.50000    0.00000
  81   0  1.5     1   2    19        3.155   -0.86464    0.50239    0.00000
  82   0  1.5     1   2    21        3.155   -0.86464    0.50239    0.00000
  83   0  2.5    -1   2    19        7.903   -0.81678    0.57694    0.00000
  84   0  2.5    -1   2    21        7.903   -0.81678    0.57694    0.00000
  85   0  2.5    -1   2    23        7.903   -0.81678    0.57694    0.00000
  86   0  2.5     1   2    18        7.763   -0.81500    0.57946    0.00000
  87   0  2.5     1   2    20        7.763   -0.81500    0.57946    0.00000
  88   0  2.5     1   2    22        7.763   -0.81500    0.57946    0.00000
  89   0  3.5    -1   2    18       13.909   -0.79105    0.61175    0.00000
  90   0  3.5    -1   2    20       13.909   -0.79105    0.61175    0.00000
  91   0  3.5    -1   2    22       13.909   -0.79105    0.61175    0.00000
  92   0  3.5    -1   2    24       13.909   -0.79105    0.61175    0.00000
  93   0  3.5     1   2    17       13.684   -0.78904    0.61435    0.00000
  94   0  3.5     1   2    19       13.684   -0.78904    0.61435    0.00000
  95   0  3.5     1   2    21       13.684   -0.78904    0.61435    0.00000
  96   0  3.5     1   2    23       13.684   -0.78904    0.61435    0.00000
  97   0  4.5    -1   2    17       21.054   -0.77523    0.63168    0.00000
  98   0  4.5    -1   2    19       21.054   -0.77523    0.63168    0.00000
  99   0  4.5    -1   2    21       21.054   -0.77523    0.63168    0.00000
 100   0  4.5    -1   2    23       21.054   -0.77523    0.63168    0.00000
 101   0  4.5    -1   2    25       21.054   -0.77523    0.63168    0.00000
 102   0  4.5     1   2    16       20.724   -0.77301    0.63439    0.00000
 103   0  4.5     1   2    18       20.724   -0.77301    0.63439    0.00000
 104   0  4.5     1   2    20       20.724   -0.77301    0.63439    0.00000
 105   0  4.5     1   2    22       20.724   -0.77301    0.63439    0.00000
 106   0  4.5     1   2    24       20.724   -0.77301    0.63439    0.00000
 107   0  5.5    -1   2    16       29.099   -0.76452    0.64460    0.00000
 108   0  5.5    -1   2    18       29.099   -0.76452    0.64460    0.00000
 109   0  5.5    -1   2    20       29.099   -0.76452    0.64460    0.00000
 110   0  5.5    -1   2    22       29.099   -0.76452    0.64460    0.00000
 111   0  5.5    -1   2    24       29.099   -0.76452    0.64460    0.00000
 112   0  5.5    -1   2    26       29.099   -0.76452    0.64460    0.00000
 113   0  5.5     1   2    15       28.644   -0.76209    0.64747    0.00000
 114   0  5.5     1   2    17       28.644   -0.76209    0.64747    0.00000
 115   0  5.5     1   2    19       28.644   -0.76209    0.64747    0.00000
 116   0  5.5     1   2    21       28.644   -0.76209    0.64747    0.00000
 117   0  5.5     1   2    23       28.644   -0.76209    0.64747    0.00000
 118   0  5.5     1   2    25       28.644   -0.76209    0.64747    0.00000
ITERM=  1 ILAM=  4 LAMBDA=  1 MU=  0 LAMNUM(ILAM)=      75
ITERM=  2 ILAM=  4 LAMBDA=  1 MU= -1 LAMNUM(ILAM)=     448
ITERM=  3 ILAM=  4 LAMBDA=  1 MU= -1 LAMNUM(ILAM)=     448
ITERM=  4 ILAM=  4 LAMBDA=  1 MU= -1 LAMNUM(ILAM)=     783
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS      783
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    2.0 %
      CURRENT DATE:  Jan 24 14:45:15 2023
 ===============================================================================
